app/views/dashboards/following_tags.html.erb

```diff
@@ -54,3 +60,36 @@
     </div>
   </div>
 </main>
+
+<script>
+  document.getElementById('follows_update_form').addEventListener('submit', checkChanged)
+  
+  function addChanged(element) {
+    element.setAttribute('changed', true);
+  };
+
+  function checkChanged(event) {
+    if (document.querySelector('input[changed]')) {
+      disableAllUnchanged();
+    }
+    else {
+      event.preventDefault();
+    }
+  };
```

Comment:
**non-blocking, nitpick**

I think this is a neat idea! I personally like having the button be `disabled` too if nothing is changed. Maybe you can add `button.disabled = true` in the `else`?

Reply:
Thanks! But if we disable the button in the `else`, because the form submission is stopped by `preventDefault()`, all the buttons will be disabled and the user won't have access to change them until refreshing the page.